src/global.d.ts

```diff
@@ -0,0 +1 @@
+declare module "reactjs-tag-input";
```

Comment:
Test the changes requested for the package.json, if the changes work this file should also be removed